pkgs/servers/x11/xorg/xwayland.nix

```diff
@@ -1,30 +1,27 @@
 { lib, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config
-, epoxy, libxslt, libunwind, makeWrapper, egl-wayland
-, defaultFontPath ? "" }:
+, epoxy, libtirpc, libxslt, libunwind, makeWrapper, egl-wayland, fetchurl, meson, ninja
+, pkg-config, fontutil, defaultFontPath ? "" }:
 
 with lib;
 
 xorgserver.overrideAttrs (oldAttrs: {
 
-  name = "xwayland-${xorgserver.version}";
-  buildInputs = oldAttrs.buildInputs ++ [ egl-wayland ];
+  name = "xwayland-21.0.99.902";
```

Comment:
Since this is still using `xorgserver.overrideAttrs` and name is set in the original `xorgserver` derivation, this is not possible here. Question is if we really still need this and can instead turn xwayland into its own derivation.

Reply:
> Question is if we really still need this and can instead turn xwayland into its own derivation.

That would probably be preferable if we overwrite almost all things anyway.